frontend/packages/erd-core/src/features/erd/components/ERDRenderer/CommandPalette/CommandPalette.module.css

```diff
@@ -6,24 +6,38 @@
   width: 638px;
   border-radius: var(--border-radius-lg);
   box-shadow: 0px 4px 20px 0px var(--shadow-basic-shadow, #000);
-  animation: contentShow 150ms cubic-bezier(0.16, 1, 0.3, 1);
   background-color: var(--pane-background);
   color: var(--global-foreground);
   border: 1px solid var(--global-border);
   z-index: var(--z-index-command-palette);
   font-family: var(--main-font);
 }
 
+.content[data-state='open'] {
+  animation: contentShow 150ms cubic-bezier(0.16, 1, 0.3, 1);
+}
+
+.content[data-state='closed'] {
```

Comment:
I follow this document for closing animation.
https://www.radix-ui.com/primitives/docs/guides/animation